contracts/hooks/examples/LimitOrder.sol

```diff
@@ -57,7 +57,7 @@ contract LimitOrder is BaseHook {
 
     bytes internal constant ZERO_BYTES = bytes("");
 
-    Epoch private constant EPOCH_DEFAULT = Epoch.wrap(0);
+    Epoch private constant EPOCH_EMPTY = Epoch.wrap(0);
```

Comment:
yeah i dont mind this - but i'd maybe go for `ZERO_EPOCH`? to match when we use things like `ZERO_ADDRESS`?